dll/win32/wshtcpip/iflist.c

```diff
@@ -8,9 +8,9 @@
 
 #include "wshtcpip.h"
 
-#define WIN32_NO_STATUS   /* Tell Windows headers you'll use ntstatus.s from NDK */
-#include <windows.h>      /* Declare Windows Headers like you normally would */
-#include <ntndk.h>        /* Declare the NDK Headers */
+//#define WIN32_NO_STATUS
+//#include <windows.h>
+//#include <ntndk.h>
```

Comment:
Any reason to just comment it, instead of stripping it entirely? I mean, after all we do have a version control system. We can always see how something looked in the past. And we are good at performing regression-tests. I think keeping dead code is not a good idea in 99.9% of the times.

Reply:
I wanted to test which of these headers were unnecessary (given also the inclusion of the "main" wshtcpip.h header above), _freely_ by having the github bots run on this.